src/llm/response_parsers/utils.hpp

```diff
@@ -0,0 +1,38 @@
+//*****************************************************************************
+// Copyright 2025 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+#include <random>
+#include <string>
+
+namespace ovms {
+std::string generateRandomId() {
+    static const char alphanum[] =
+        "0123456789"
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        "abcdefghijklmnopqrstuvwxyz";
+    static constexpr int idLength = 9;
+    static thread_local std::mt19937 rng{std::random_device{}()};
+    static thread_local std::uniform_int_distribution<> dist(0, sizeof(alphanum) - 2);
+
+    std::string id;
```

Comment:
Please verify, if we can rely on this id randomizer from SDL standpoint

Reply:
what latency is adding this function? I wonder if a sequential number would be quicker and good enough.